ldes-server-port-ingestion-rest/src/main/java/be/vlaanderen/informatievlaanderen/ldes/server/ingestion/rest/LdesMemberIngestionController.java

```diff
@@ -5,19 +5,19 @@
 import org.springframework.web.bind.annotation.RestController;
 
 import be.vlaanderen.informatievlaanderen.ldes.server.domain.entities.LdesMember;
-import be.vlaanderen.informatievlaanderen.ldes.server.domain.services.SdsReader;
+import be.vlaanderen.informatievlaanderen.ldes.server.domain.services.LdesReader;
 
 @RestController
 public class LdesMemberIngestionController {
 
-    private final SdsReader sdsReader;
+    private final LdesReader ldesReader;
 
-    public LdesMemberIngestionController(final SdsReader sdsReader) {
-        this.sdsReader = sdsReader;
+    public LdesMemberIngestionController(final LdesReader ldesReader) {
+        this.ldesReader = ldesReader;
     }
 
     @PostMapping(value = "/ldes-member", consumes = "application/n-quads")
```

Comment:
In the future, should not specify a content type, but instead try to determine it automaticallly.